lib/_http_client.js

```diff
@@ -370,6 +377,10 @@ ClientRequest.prototype._finish = function _finish() {
       request: this,
     });
   }
+  if (isTraceHTTPCategory()) {
+    this._traceEventId = getNextTraceEventId();
+    traceBegin('http.client.request', this._traceEventId);
```

Comment:
Can we move this to a constant value? Since we need to sync the naming on line 668.

Reply:
Thanks ! Done !